kiali-server/templates/role.yaml

```
@@ -99,4 +99,10 @@ rules:
   - list
   - patch
   - watch
+- apiGroups:
+  - authentication.k8s.io
```

Maintain consistency with the rest of this file - specify apiGroups using `[ ]` notation.
```suggestion
- apiGroups: ["authentication.k8s.io"]
```

---

done